meta/main.yml

```diff
@@ -16,9 +16,6 @@ galaxy_info:
     - 7
   categories:
   - system
-dependencies: []
-  # List your role dependencies here, one per line. Only
-  # dependencies available via galaxy should be listed here.
-  # Be sure to remove the '[]' above if you add dependencies
-  # to this list.
-  
+dependencies:
+  - role: tor
+    when: "enable_onion_support == True"
```

Comment:
So I supposed that it is, for type reason ( ie, enable_onion_support not being a boolean, etc). Now, I suspect for the use case we want, it would work fine, but i need to test.